src/relayServerUtils.ts

```diff
@@ -332,6 +332,30 @@ async function convertGasToTokenAndNative(
   };
 }
 
+function secondsToDate(dateInSeconds: number) {
+  return new Date(dateInSeconds * 1000);
+}
+
+async function validateExpirationTime(
+  validUntilTime: PromiseOrValue<BigNumberish>,
+  requestMinValidSeconds: number
+) {
+  const validUntilTimeValue = await validUntilTime;
+  const secondsNow = Math.round(Date.now() / 1000);
+  const expiredInSeconds =
+    parseInt(validUntilTimeValue.toString()) - secondsNow;
```

Comment:
```suggestion
    parseInt(validUntilTime.toString()) - secondsNow;
```

Reply:
Why do we want to remove line 343? The value we receive there is of type `PromiseOrValue`. Although we usually have a value there, it's safe to await for it in case we received a Promise

Reply:
We discussed about it.